### DISCUSSION TITLE
Code Tips

### DISCUSSION THREAD

**Discussion by franticxx** [zed-industries/zed#6486] [Feature Requests]
Why is there no code hint in my zed, I have enabled enable_language_server and lsp in the Settings, and rust-analysis has been downloaded automatically

**Comment by imag1ne** [comment]
Do you mean the inlay hints? I don't see them either. But I think you can see them from hover popovers.

**Comment by JosephTLyons** [comment]
We currently don't have inlay hints yet, but it is a popular request.  Could you upvote this issue?

- https://github.com/zed-industries/community/issues/138

In the meanwhile, as @imag1ne said, you can get the type information by hovering on the symbol.

**Comment by JosephTLyons** [comment]
Just to update this older Discussion - Zed does support inlay hints now:

<img width="1139" alt="SCR-20240528-tlch" src="https://github.com/zed-industries/zed/assets/19867440/bc518f32-4037-4f5d-a39f-83eb1703c4ab">

Note you might need to add some custom settings to make it work for your language - some languages are documented [here](https://zed.dev/docs/configuring-zed#inlay-hints).
